### PR TITLE
Add tour managing for admin

### PR DESCRIPTION
**Pull request by danhuynhdev**
image

### CODE REVIEW

**Comment by danhuynhdev** [comment]
ready

File: app/controllers/admin/tours_controller.rb [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      if @tour.update_attributes tour_params
+        flash[:success] = t "shared.success_messages.changes_saved"
+        redirect_to admin_tour_url(@tour)
+      else
+        render :edit
+      end
+    end
+
+    def destroy
+      @tour.destroy
```
Comment (nvtanh) [inline app/controllers/admin/tours_controller.rb]: nếu xóa ko dc thì sao e ?

**Comment by nvtanh** [comment]
commented

**Comment by danhuynhdev** [comment]
ready

**Comment by tranducquoc** [comment]
merged

**Pull request merged**